docs/data_sources/eia860.rst

```diff
@@ -88,4 +88,16 @@ Where yyyy stands for a given year. Ex: LayoutYyyy = LayoutY2013 or LayoutY2014
 PUDL Data Transformations
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The
+PUDL Data Field Descriptions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+:ref:`generators_eia860`
+
+:ref:`ownership_eia860`
+
+:ref:`boiler_generator_assn_eia860`
+
+:ref:`plants_eia860`
+
+:ref:`utilities_eia860`
+
+:ref:`generators_eia860`
```

Comment:
I wonder if there's an easy way we could generate these references automatically, so that we don't ever forget to add new tables, or leave them out accidentally.

Reply:
so far, I can't find anything :/